src/hooks/mutations/useAdminGetApprovedRequests.jsx

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from "@tanstack/react-query";
```

Comment:
I don't think you're using this file any more. You can go ahead to delete this file.